ui/src/app/features/delete-dialog/delete-dialog.component.ts

```diff
@@ -23,11 +23,14 @@ import { MAT_DIALOG_DATA, MatDialogRef } from '@angular/material/dialog';
 })
 
 export class DeleteDialogComponent implements OnInit {
+  isOrganizationOwner: boolean;
```

Comment:
this component is general, move this logic to the place where dialog is called, and pass isOrganizationOwner property with data